packages/eventual-send/src/types.d.ts

```diff
@@ -17,7 +17,7 @@ export type * from './track-turns.js';
 //   Types exposed to modules.
 //
 
-export type Callable = (...args: unknown[]) => any;
+export type Callable = (...args: any[]) => any;
```

Comment:
Thank you!  This kept flipping back and forth so much that I lost track of the fact that `unknown[]` broke the typing.